dts/bindings/pinctrl/nxp,s32ze-pinctrl.yaml

```diff
@@ -116,3 +116,11 @@ child-binding:
             5: FMAX_33 = 50 MHz
             6: FMAX_33 = 50 MHz
             7: FMAX_33 = 1 MHz
+
+      current-reference-control:
+        type: boolean
+        description: Applies current reference control to the associated pin
```

Comment:
at least this should have a note that this configuration is only applicable for LVDS pads and has no effect on other pad types. Same for `termination-resistor`.

Reply:
I updated